blocks/donation-form/class-give-donation-form-block.php

```diff
@@ -149,6 +150,37 @@ public function render_donation_form( $attributes ) {
 
 		return give_form_shortcode( $parameters );
 	}
+
+	/**
+	 * This function is used to fetch donation forms based on the chosen search in the donation form block.
+	 *
+	 * @since  2.5.3
+	 * @access public
+	 *
+	 * @return void
+	 */
+	public function block_donation_form_search_results() {
```

Comment:
@mehul0810 did you check existing WP endpoint to get custom post type search results?

Reply:
@ravinderk This is not possible as of now with `WithSelect` function. So, I'll circle back on this point in future. So, I've implemented the custom AJAX logic here.